docs/src/components/Layout/Page/Page.component.tsx

```diff
@@ -204,7 +208,7 @@ Components.strong = (props: any) => (
 );
 Components.ul = (props: any) => <ul {...props} />;
 Components.ol = (props: any) => <ol {...props} />;
-Components.li = (props: any) => <li {...props} />;
+Components.li = (props: any) => <StyledLi><Typography {...props} /></StyledLi>;
```

Comment:
This can just be `<Typography mb={1}  />` can't it?

Reply:
The component that is returned needs to be wrapped in an <li> tag, so at a minimum what would have to be returned is `<li><Typography mb={1}...`. That said, because what I'm trying to pad is the container holding the content I'd prefer putting the padding on the parent rather than the child.

Reply:
👍